net.rocketpool/image/types/ethers-contracts/RethAbi.ts

```diff
@@ -0,0 +1,791 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Can we add a README file with quick instructions on how this files are generated?

Reply:
Done